glean-core/src/core/mod.rs

```diff
@@ -697,6 +705,22 @@ impl Glean {
         metric.test_get_value(self)
     }
 
+    /// Set configuration for metrics' disabled property, typically from a Nimbus experiment
+    /// or rollout
+    ///
+    /// # Arguments
+    ///
+    /// * `json` - The stringified JSON representation of a MetricsDisabled object
+    pub fn set_metrics_disabled_config(&self, json: String) -> Result<()> {
```

Comment:
Maybe this should take `cfg: MetricsDisabledConfig` instead and the outer function do the parsing?